src/cli-commands/update-dependencies.js

```diff
@@ -0,0 +1,98 @@
+const fs = require('fs-extra');
+const path = require('path');
+const spawn = require('cross-spawn');
+const logger = require('@blackbaud/skyux-logger');
+const latestVersion = require('latest-version');
+const rootPath = process.cwd();
+const rimraf = require('rimraf');
```

Comment:
Need to add `rimraf` as a dependency in package.json.

Reply:
done :) darn devDependencies tricked me